bottlecap/src/metrics/enhanced/lambda.rs

```diff
@@ -1,9 +1,12 @@
-use crate::metrics::enhanced::{
-    constants::{self, BASE_LAMBDA_INVOCATION_PRICE},
-    statfs::statfs_info,
-};
 use crate::proc::{self, CPUData, NetworkData};
 use crate::telemetry::events::ReportMetrics;
+use crate::{
```

Comment:
Might be good to either merge imports, or, just pass the telemetry import to the one above